server/main.py

```diff
@@ -26,29 +26,44 @@
 DECREASE_SPEED = 0x0F
 TERMINATE = 0x10
 
-COMMANDS_CENTRAL = {
-    PING: DRONE.ping,
-    TAKE_OFF: DRONE.takeoff,
-    LAND: DRONE.land,
-    HOVER: DRONE.hover,
-    MOVE_LEFT: DRONE.move_left,
-    MOVE_RIGHT: DRONE.move_right,
-    MOVE_UP: DRONE.move_up,
-    MOVE_DOWN: DRONE.move_down,
-    MOVE_FORWARD: DRONE.move_forward,
-    MOVE_BACKWARD: DRONE.move_backward,
-    TURN_LEFT: DRONE.turn_left,
-    TURN_RIGHT: DRONE.turn_right,
-    RESET: DRONE.reset,
-    CALIBRATE: DRONE.trim,
-    INCREASE_SPEED: DRONE.increase_speed,
-    DECREASE_SPEED: DRONE.decrease_speed,
-    TERMINATE: DRONE.halt
+
+def main_init():
+    global DRONE, COMMANDS_CENTRAL
+    DRONE = libardrone.ARDrone()
```

Comment:
Setting global variables is a big no-no in Python... I think we can do everything we need without using global variables.